### ISSUE TITLE
Avoid redefining n()

### ISSUE THREAD

**Issue by krlmlr** [randomchars42/bioset#1]
The release candidate of dplyr 0.7.5 will use the correct implementation of `n()` only if this function isn't overridden. Most likely because of this, CRAN checks of bioset fail now: https://github.com/tidyverse/dplyr/blob/master/revdep/new-problems.md#bioset

A solution seems to be to assign `dplyr::n` to the local variant of `n`, or to avoid overriding `n`.

**Comment by randomchars42** [comment]
Hey krlmlr,

if I undestand you correctly I need to add to the following existing code

```
    data <- data %>%
      dplyr::group_by(!! ids) %>%
      dplyr::mutate(
        !! target_n := n(),
        !! target_mean := mean(!! target),
        !! target_sd := stats::sd(!! target),
        !! target_cv := stats::sd(!! target) / mean(!! target)
      ) %>%
      dplyr::ungroup()
```
something like:

```
n <- dplyr::n
```

?

I'm not aware that I redefine / override `n()` anywhere explicitly or implicitly, would you please be so kind as to explain what might trigger this new error?

**Comment by krlmlr** [comment]
Yes. Either this, or just import `n()` from *dplyr*, or use `dplyr::n()` (the latte requires *dplyr* 0.7.5).

https://github.com/randomchars42/bioset/blob/76a755e3d12c08248d087cf691c093180a275400/R/sets.R#L367

**Comment by krlmlr** [comment]
We now double-check if e.g. `mean()` still means `base::mean()` or something else. If not, we don't use hybrid evaluation. Because *bioset* overrides `n()`, we call the custom version instead of the built-in implementation.

Earlier *dplyr* didn't export `n()`, so this might have been necessary to avoid CRAN check errors.

**Comment by randomchars42** [comment]
Oh..., Thanks! I obviously forgot about that line of code.... Thanks for the kind help and illuminating explanation! I've adjusted the code and it (hopefully) is on its way to CRAN now.
